demo/launch/constrained_planning_demo.launch.py

```diff
@@ -0,0 +1,152 @@
+import os
+from launch import LaunchDescription
+from launch_ros.actions import Node
+from launch.actions import ExecuteProcess
+from ament_index_python.packages import get_package_share_directory
+from moveit_configs_utils import MoveItConfigsBuilder
+
+
+def generate_launch_description():
+    moveit_config = (
+        MoveItConfigsBuilder("moveit_resources_panda")
+        .robot_description(file_path="config/panda.urdf.xacro")
+        .trajectory_execution(file_path="config/gripper_moveit_controllers.yaml")
+        .planning_pipelines(pipelines=["ompl"])
+        .to_moveit_configs()
+    )
+
+    # Set planning pipeline parameters
+    moveit_config.planning_pipelines["ompl"]["panda_arm"][
+        "enforce_constrained_state_space"
+    ] = True
+    moveit_config.planning_pipelines["ompl"]["panda_arm"][
+        "projection_evaluator"
+    ] = "joints(panda_joint1,panda_joint2)"
+    moveit_config.planning_pipelines["ompl"]["panda_arm_hand"][
+        "enforce_constrained_state_space"
+    ] = True
+    moveit_config.planning_pipelines["ompl"]["panda_arm_hand"][
+        "projection_evaluator"
+    ] = "joints(panda_joint1,panda_joint2)"
+
+    drake_ktopt_planning_pipeline_config = {
+        "drake_ktopt": {
+            "planning_plugins": [
+                "ktopt_interface/KTOptPlanner",
+            ],
+            "request_adapters": [
+                "default_planning_request_adapters/ResolveConstraintFrames",
+                "default_planning_request_adapters/ValidateWorkspaceBounds",
+                "default_planning_request_adapters/CheckStartStateBounds",
+                "default_planning_request_adapters/CheckStartStateCollision",
+            ],
+            "response_adapters": [
+                "default_planning_response_adapters/ValidateSolution",
+                "default_planning_response_adapters/DisplayMotionPath",
+            ],
+        }
+    }
+
+    # Start the actual move_group node/action server
+    run_move_group_node = Node(
+        package="moveit_ros_move_group",
+        executable="move_group",
+        output="screen",
+        parameters=[moveit_config.to_dict(), drake_ktopt_planning_pipeline_config],
```

Comment:
This doesn't work yet. @kamiradi I won't be able to look further into it, maybe you can fix it. Feel free to just push to this PR